Vic2ToHoI4/Source/Hoi4Outputter/GameRules/OutGameRule.cpp

```diff
@@ -0,0 +1,25 @@
+#include "OutGameRule.h"
+
+
+
+std::ostream& HoI4::operator<<(std::ostream& output, const HoI4::GameRule& rule)
+{
+	output << rule.key << " = {\n";
+	output << "\tname = \"" << rule.name << "\"\n";
+	if (rule.requiredDlc)
```

Comment:
curiosity, are there any rules requiring multiple DLCs?

Reply:
Not currently. Of course, if a later HoI4 patch changes that, we'll need to update to match. But for now I'll with the good old YAGNI rule.